setup.sh

```diff
@@ -14,4 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-pip3 install -r requirements.txt -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
\ No newline at end of file
+# Uninstall existing JAX and JAXlib
+pip3 show jax && pip3 uninstall -y jax 
+pip3 show jaxlib && pip3 uninstall -y jaxlib
+
+# Install dependencies from requirements.txt
+pip3 install -r requirements.txt -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

Comment:
Not sure I understand this. We install `requirements.txt` -- but doesn't taht still contain `jax[tpu]`? Shouldn't we remove `jax[tpu]` from requirements.txt and only install it if necessary?